meeting-minutes/2025/2025-06-18.md

```diff
@@ -0,0 +1,145 @@
+# 1. Opening Activities
+
+## 1.1 Opening comments (Co-Chair)
+
+- Justin & Omar open the meeting.
+
+## 1.2 Introduction of participants/roll call (Co-Chair)
+
+Quorum requires participation of 8 or more of the 14 voting members (including the officers).
+
+| First Name | Last Name   | Company                                                     | Role(s)       | Present |
+|:-----------|:------------|:------------------------------------------------------------|:--------------|:--------|
+| Adrian     | Diglio      | Microsoft                                                   | Member        | No      |
+| Art        | Middlekauff | Flexera                                                     | Voting Member | No      |
+| David      | Kemp        | National Security Agency                                    | Member        | No      |
+| Denny      | Page        | Individual                                                  | Voting Member | No      |
+| Feng       | Cao         | Oracle                                                      | Voting Member | Yes     |
+| Harin      | Sarda       | Cisco Systems                                               | Voting Member | Yes     |
+| Jautau     | White       | Microsoft                                                   | Voting Member | Yes     |
+| Jeremy     | Rickard     | Microsoft                                                   | Member        | No      |
+| Justin     | Murphy      | DHS Cybersecurity and Infrastructure Security Agency (CISA) | Co-Chair      | Yes     |
+| Kris       | Vandecruys  | Cisco Systems                                               | Member        | No      |
+| Kunal      | Modasiya    | Qualys, Inc.                                                | Member        | No      |
+| Langley    | Rock        | Dell                                                        | Voting Member | Yes     |
+| Martin     | Prpic       | Red Hat                                                     | Member        | No      |
+| Omar       | Santos      | Cisco Systems                                               | Co-Chair      | Yes     |
+| Pablo      | Quiroga     | Qualys, Inc.                                                | Member        | No      |
+| Peter      | Gephardt    | IBM                                                         | Member        | No      |
+| Przemyslaw | Roguski     | Red Hat                                                     | Voting Member | No      |
+| Sergii     | Demianchuk  | Cisco Systems                                               | Member        | No      |
+| Shridhar   | Chari       | Cisco Systems                                               | Voting Member | No      |
+| Sonny      | van Lingen  | Huawei Technologies Co., Ltd.                               | Voting Member | Yes     |
+| Stefan     | Arntzen     | Huawei Technologies Co., Ltd.                               | Voting Member | No      |
```

Comment:
According to [May Meeting Minutes](https://github.com/oasis-tcs/openeox/blob/main/meeting-minutes/2025/2025-05-21.md), Stefan lost voting rights last meeting.
```suggestion
| Stefan     | Arntzen     | Huawei Technologies Co., Ltd.                               | Member | No      |
```